### ISSUE TITLE
Loosing control in MBT without mouse

### ISSUE THREAD

**Issue by sfnetwork** [MediaBrowser/Emby.Theater.Windows#17]
MBT primary control devices should be keyboard or remote (not mouse). So all of my testing are made using keyboard only.

Arrow keys should bring back control in any situation.
In this example, I simply clicked where the arrow is pointing:
![2013-10-12 1-39-34 pm](https://f.cloud.github.com/assets/5288374/1320563/5bd2aa54-3365-11e3-8a48-b026124ae48e.jpg)

After this point, without a mouse, there is no way to select anything with keyboard...

There are other scenarios besides clicking in blank spaces that reproduce this issue.
This would be an issue on a dedicated HTPC.
